.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.15.x, 1.16.x]
+        go-version: [1.15.x, 1.16.x, 1.17.x]
```

Comment:
Thanks :+1:  Could you also remove 1.15.x?

Reply:
Done